content/docs/client-options.md

```diff
@@ -30,6 +30,7 @@ These clients are compatible with our [staging endpoint for ACME v2](https://com
 - [acme.sh](https://github.com/Neilpang/acme.sh/tree/2) (`2` branch)
 - [Net::ACME2](https://metacpan.org/pod/Net::ACME2)
 - [EasyHTTPs](https://easy.zhetao.com) * (Automatically select v2 or v1)
+- [Let's Encrypt for Hiawatha](https://www.hiawatha-webserver.org/files/letsencrypt.tar.gz)
```

Comment:
Can you link to a webpage that describes the V2 support and where to download this `.tar.gz` instead?

As-is this will be the only link on the client options page that instead of bringing you to a website, immediately initiates a download. I think it would be a better experience for curious users if you could add a link to a website with a download link instead.